content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,125 @@
+
+## Maneiras de se conectar
+
+Você tem várias opções para se conectar a nós, pods e serviços de fora do cluster:
+
+  - Acesse serviços através de IPs públicos.
+    - Use um serviço com tipo `NodePort` ou `LoadBalancer` para tornar o serviço acessível fora do cluster. Consulte a documentação de [serviços]((/docs/concepts/services-networking/service/)) e
+[kubectl expose](/docs/reference/generated/kubectl/kubectl-commands/#expose).
+    - Dependendo do ambiente do cluster, isso pode expor o serviço apenas para a rede corporativa, ou pode expô-lo para a Internet. Pense se o serviço que está sendo exposto é seguro. Ele faz sua própria autenticação?
+    - Coloque pods atrás de serviços. Para acessar um pod específico de um conjunto de réplicas, como para depurar, coloque um rótulo exclusivo no pod e crie um novo serviço que selecione este rótulo.
+    - Na maioria dos casos, não deve ser necessário para o desenvolvedor de aplicativos acessar diretamente nós através de seus IPs de nó.
+  - Acesse serviços, nós ou pods usando o Verbo Proxy.
+    - Faz autenticação e autorização do servidor de API antes de acessar o serviço remoto. Use isto se os serviços não forem seguros o suficiente para expor à Internet, ou para obter acesso a portas no IP do nó, ou para depuração.
+    - Proxies podem causar problemas para algumas aplicações web.
+    - Só funciona para HTTP/HTTPS.
+    - Descrito [aqui](#manually-constructing-apiserver-proxy-urls).
+  - Acesse a partir de um nó ou pod no cluster.
+    - Execute um pod e, em seguida, conecte-se a um shell nele usando [kubectl exec](/docs/reference/generated/kubectl/kubectl-commands/#exec). Conecte-se a outros nós, pods e serviços a partir desse shell.
+    - Alguns clusters podem permitir que você faça ssh para um nó no cluster. De lá, você pode ser capaz de acessar os serviços do cluster. Este é um método não padrão e funcionará em alguns clusters, mas
+não em outros. Navegadores e outras ferramentas podem ou não estar instalados. O DNS do cluster pode não funcionar.
+
+## Descobrindo serviços integrados
+
+Normalmente, existem vários serviços que são iniciados em um cluster pelo kube-system. Obtenha uma lista desses serviços com o comando `kubectl cluster-info`:
+
+```shell
+kubectl cluster-info
+```
+
+A saída é semelhante a esta:
+
+```
+O mestre do Kubernetes está em execução em https://192.0.2.1
+elasticsearch-logging está em execução em https://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy
+kibana-logging está em execução em https://192.0.2.1/api/v1/namespaces/kube-system/services/kibana-logging/proxy
+kube-dns está em execução em https://192.0.2.1/api/v1/namespaces/kube-system/services/kube-dns/proxy
+grafana está em execução em https://192.0.2.1/api/v1/namespaces/kube-system/services/monitoring-grafana/proxy
+heapster está em execução em https://192.0.2.1/api/v1/namespaces/kube-system/services/monitoring-heapster/proxy
```

Comment:
I liked it "Mestre do Kubernetes", but command output it is better keep the original output.